magniv/export/airflow/export_to_airflow.py

```diff
@@ -40,8 +40,8 @@ def export_to_airflow(
         print(new_filename)
         if not os.path.exists("dags/"):
             os.mkdir("dags")
-        if not os.path.exists("dags/{}/".format(task_info["owner"])):
-            os.mkdir("dags/{}/".format(task_info["owner"]))
+        if not os.path.exists(f'dags/{task_info["owner"]}/'):
+            os.mkdir(f'dags/{task_info["owner"]}/')
```

Comment:
if using format strings (or whatever this "f" is) considered better practice?

Reply:
yes it's more readable, flexible and generally faster https://stackoverflow.com/questions/43123408/f-strings-vs-str-format

Reply:
oh so these are the speed improvements?

Reply:
lol ya and getting rid of nested if statements, perhaps not best word choice